build.gradle

```diff
@@ -139,6 +164,72 @@ allprojects {
     }
 }
 
+def log4jVersion = '2.22.1'
+
+configurations {
+    distOnly
+}
+
+dependencies {
+    distOnly "org.apache.logging.log4j:log4j-core:${log4jVersion}"
+    distOnly "org.apache.logging.log4j:log4j-slf4j2-impl:${log4jVersion}"
+}
+
+tasks.register('dist') {
+    group = JavaBasePlugin.BUILD_TASK_NAME
+    description "Creates the distribution directory"
```

Comment:
The value of `JavaBasePlugin.BUILD_TASK_NAME` is `build`, which is a different meaning comparing to the `Creates the distribution directory` description.

I don't think these two lines are particularly necessary, since the task name of the dist task itself already accounts for its meaning.

Reply:
Removed.